docpages/example_programs/the_basics/editing-channels-and-messages.md

```diff
@@ -1,15 +1,23 @@
-\page editing-channels-and-messages Editing Channels and Messages
-
-Sometimes we need to update an object, such as message or channel. At first, it might seem confusing, but it's actually really simple! You just need to use an object with identical properties you don't need to update. NOTE: your bot can't edit messages sent by others.
-
-\note This example uses callback functions. To see more information about them, visit \ref callback-functions.
-
-\include{cpp} editing_messages.cpp
-
-Before editing:
-
-\image html stuff_edit1.png
-
-After editing:
-
-\image html stuff_edit2.png
\ No newline at end of file
+\page editing-channels-and-messages Editing Channels and Messages
+
+Sometimes we need to update an object, such as a message (whether it's plain text or an embed) or a channel. At first, it might seem confusing, but it's actually really simple! You need an object with all of the properties being identical to the existing one. Say you're editing a message. You need to have an object identical to the one in Discord. Then you replace what you need such as its content. NOTE: your bot can't edit messages sent by others.

```

Comment:
Stick the `NOTE:` in an actual `\note`.

`You need to have an object identical to the one in Discord` I feel like this is misleading, the only thing that needs to match is the ID, every other field can be changed (which is what you do to edit it).

Reply:
Btw, you don't have to correct the `You need to have an object identical to the one in Discord` part here right now, would like to see what others think.

Reply:
> Stick the `NOTE:` in an actual `\note`.

Should I move the references to other pages into another line? How do I do so while keeping them in the same note block?

Reply:
just do two separate note blocks

Reply:
ok